jx/JXML.js

```diff
@@ -30,13 +37,18 @@ JXML.import = function(name, cb) {
  * `args` will be sanitized through JSON.
  */
 JXML.cast = function(uid, method, args) {
-	args = JSON.parse(JSON.stringify(args || {}));
+	var array_args = [];
+
+	for (var i = 0; i in args; i++)
```

Comment:
`args` comes in as an object because all arrays became objects in `setAttr({ onclick: [...]})`. Maybe we should move this conversion into handle_cast in the future, or have local functions accept a single object from the mailbox.